delfin/common/constants.py

```diff
@@ -148,6 +148,49 @@ class DiskLogicalType(object):
     ALL = (FREE, MEMBER, HOTSPARE, CACHE, UNKNOWN)
 
 
+class FilesystemStatus(object):
+    NORMAL = 'normal'
+    FAULTY = 'faulty'
+
+    ALL = (NORMAL, FAULTY)
+
+
+class WORMType(object):
```

Comment:
Not very clear as WORM is basically compliance or protection. Can we call it WORMProtection?